docs/develprocess.md

```diff
@@ -650,12 +650,17 @@ bunzip2 < ../install.img | cpio -idm
 cd ..
 ```
 
-### Navigate in the installer's filesystem
+#### Navigate in the installer's filesystem
 
 If you want to use commands in the installer's filesystem context, as root:
 ```
 chroot install/
 ```
+You'll also need to mount `urandom` in your chrooted dir:
```

Comment:
Let's explain that the commands below have to be run from outside the chroot and that it's needed for using `rpm` and `yum` commands?